BlueprintUI/Sources/Layout/Column.swift

```diff
@@ -43,13 +43,46 @@ public struct Column: StackElement {
 
     public var children: [(element: Element, traits: StackLayout.Traits, key: AnyHashable?)] = []
 
-    public private(set) var layout = StackLayout(
-        axis: .vertical,
-        alignment: ColumnAlignment.leading.stackAlignment
-    )
+    public private(set) var layout: StackLayout = .defaultColumn
 
     public init() {}
 
+    /// Initializer using result builder to declaritively build up a stack.
+    /// - Parameters:
+    ///   - layout: A StackLayout describing the column's layout
+    ///   - elements: A block containing all elements to be included in the stack.
+    /// - Note: If element is a StackChild, then traits and key will be pulled from the element, otherwise defaults are passed through.
+    public init(
+        layout: StackLayout = .defaultColumn,
+        @ElementBuilder<StackChild> elements: () -> [StackChild]
+    ) {
+        self.init(elementsBuilder: elements)
+        self.layout = layout
+    }
+
+    /// Initializer using result builder to declaritively build up a stack.
+    /// - Parameters:
+    ///   - alignment: Specifies how children will be aligned horizontally.
+    ///   - underflow: Determines the layout when there is extra free space available.
+    ///   - overflow: Determines the layout when there is not enough space to fit all children as measured.
+    ///   - horizontalAlignment: Specifies how children will be aligned horizontally.
+    ///   - minimumSpacing: Spacing in between elements.
+    ///   - elements: A block containing all elements to be included in the stack.
+    /// - Note: If element is a plain Element, then that Element will be implicitly converted into a `StackChild` with default values
+    public init(
+        alignment: ColumnAlignment = StackLayout.defaultColumn.columnAlignment,
+        underflow: StackLayout.UnderflowDistribution = StackLayout.defaultColumn.underflow,
+        overflow: StackLayout.OverflowDistribution = StackLayout.defaultColumn.overflow,
+        minimumSpacing: CGFloat = StackLayout.defaultColumn.minimumSpacing,
```

Comment:
How do these default values show up in Xcode's autocomplete? It might be nice to put the defaults in here explicitly, even if there's some duplication, to make it easy to see which properties you need to override and which you can skip.

The doc comments should say what the defaults are, either way.

Reply:
This makes a lot of sense and lets me delete the defaultRow code 👍

Reply:
Inversely; its nice to be able to put these in one place. Hardcoding breaks that. I'd prefer leaving this as is, IMO? If someone _really_ cares; they can do look at the value.